### DISCUSSION TITLE
New unexpected guess and aws cloud

### DISCUSSION THREAD

**Discussion by bemyevent** [mozilla/hubs#2900] [Hubs Cloud Questions] [answered]
Hello 

I saw the new feature unexpected guess out of your hubs space , and want to have it in my aws setup . Is there something to do for having this update in my aws hubs ? 
Thx for the reply.

**Comment by emclaren** [comment, marked as answer]
Hi @bemyevent 

The new feature for "invite only" rooms (https://github.com/mozilla/hubs/pull/2838) will be pushed to Hubs Cloud sometime this week.

In Hubs Cloud you can also make your rooms accessible only to approved email accounts, there is more information here: https://hubs.mozilla.com/docs/hubs-cloud-limiting-user-access.html
